pointerevents/pointerevent_capture_mouse.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
I assume this is just a proof of concept test here. Right? Wasn't there a simple test already in pointerevents that only needed a move or something that you could reuse?

Reply:
Yep, this is just proof of concept. What is the test you are thinking of?

Reply:
How about this:
https://github.com/w3c/web-platform-tests/blob/master/pointerevents/pointerevent_element_haspointercapture-manual.html